inductiva/resources/storage.py

```diff
@@ -9,8 +8,8 @@ def get_space_used():
     try:
         api = instance_api.InstanceApi(inductiva.api.get_client())
         instance_price = api.get_storage_size()
-        logging.info("Total storage used: %s GB",
-                     float(round(instance_price.body, 3)))
-        return float(round(instance_price.body, 3))
+        instance_price = float(round(instance_price.body, 3))
+        print(f"Total storage used: {instance_price} GB")
```

Comment:
Can you clarify why do we need this change? 

Just worried since this also modifies the calling of the function.

Reply:
When we remove the async, that will make completely sense, since users will lose the information otherwise.